AcadeMeData/tests.py

```diff
@@ -1,35 +1,40 @@
 import pytest
-from models import User
+from AcadeMeData.models import User
 
 
-class TestUserModel(pytest):
+class TestUserModel:
     def user_example(self):
-        user_data = {'username': "username", 'password': "password", 'email': "user@example.com", 'type': "S",
-                     'university': "RU",
-                     'degree': "CS"}
-        user = User.create_user(*user_data)
-        return user, user_data
+        data_for_user = {'username': "username", 'password': "password", 'email': "user@example.com", 'type': "S",
+                         'university': "RU",
+                         'degree': "CS"}
+        user = User.create_user(*data_for_user)
+        return user
 
-    @pytest
+    @pytest.mark.django_db
     def test_create_user(self):
-        user, user_data = self.user_example()
-        assert isinstance(user, User)
-        assert user.username == user_data.get('username')
-        assert user.password == user_data.get('password')
-        assert user.email == user_data.get('email')
-        assert user.type == user_data.get('type')
-        assert user.university == user_data.get('university')
-        assert user.degree == user_data.get('degree')
-        assert User.get_user(user_data.get('username')).degree == user_data.get('degree')
+        users_list = User.objects.all()
+        user_data = self.user_example()
+        assert users_list[len(users_list) - 1].user.username == user_data.user.username

```

Comment:
It should be like that 
user_data returned from the user_example method, 
no searching in db after you have the search commend

user=User.get_user(user_data[usename])
It returns the whole object

now you have methods of User of django user

assert user.get_username() == user_data[usename]
assert user.get_email() == user_data[email]

**Note:**
you can tests the code by running the python from the /vagrant dir
pipenv run python manage.py shell
from AcadeMeData.models import User

and test every thing, tab will reveal the methods access in the classes

Reply:
This should work like this 
       user = User.create_user(*user_data)
        user = User.get_user("username")
        assert user.get_username() ==  user_data['username']


tested

Reply:
![image](https://user-images.githubusercontent.com/74950303/143479368-58af4669-9aa7-4039-8926-730bcfda88f6.png)
![image](https://user-images.githubusercontent.com/74950303/143479409-f5b1e3e8-2cbf-41f3-a362-d9298b8b7cc5.png)

this what i got. what should user_example return?
@Yarboa